integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDynamicDomainInImage.java

```diff
@@ -0,0 +1,309 @@
+  private static final String modelFile = "model-fmw-dynamicdomain.yaml";
+
+  private static String dbUrl = null;
+  private static LoggingFacade logger = null;
+
+  private String domainUid = "jrf-dynamicdomain-mii";
+  private String adminServerPodName = domainUid + "-admin-server";
+  private String managedServerPrefix = domainUid + "-managed-server";
+  private String adminSecretName = domainUid + "-weblogic-credentials";
+  private String encryptionSecretName = domainUid + "-encryptionsecret";
+  private String rcuaccessSecretName = domainUid + "-rcu-access";
+  private String opsswalletpassSecretName = domainUid + "-opss-wallet-password-secret";
+  private int replicaCount = 2;
+
+  // create standard, reusable retry/backoff policy
+  private static final ConditionFactory withStandardRetryPolicy
+      = with().pollDelay(2, SECONDS)
+      .and().with().pollInterval(10, SECONDS)
+      .atMost(5, MINUTES).await();
+
+  /**
+   * Start DB service and create RCU schema.
+   * Assigns unique namespaces for operator and domains.
+   * Pull FMW image and Oracle DB image if running tests in Kind cluster.
+   * Installs operator.
+   *
+   * @param namespaces injected by JUnit
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(3) List<String> namespaces) {
+
+    logger = getLogger();
+    logger.info("Assign a unique namespace for DB and RCU");
+    assertNotNull(namespaces.get(0), "Namespace is null");
+    dbNamespace = namespaces.get(0);
+    dbUrl = ORACLEDBURLPREFIX + dbNamespace + ORACLEDBSUFFIX;
+
+    logger.info("Assign a unique namespace for operator");
+    assertNotNull(namespaces.get(1), "Namespace is null");
+    opNamespace = namespaces.get(1);
+
+    logger.info("Assign a unique namespace for JRF domain");
+    assertNotNull(namespaces.get(2), "Namespace is null");
+    domainNamespace = namespaces.get(2);
+
+    logger.info("Start DB and create RCU schema for namespace: {0}, RCU prefix: {1}, "
+        + "dbUrl: {2}, dbImage: {3},  fmwImage: {4} ", dbNamespace, RCUSCHEMAPREFIX, dbUrl,
+        DB_IMAGE_TO_USE_IN_SPEC, FMWINFRA_IMAGE_TO_USE_IN_SPEC);
+    assertDoesNotThrow(() -> setupDBandRCUschema(DB_IMAGE_TO_USE_IN_SPEC, FMWINFRA_IMAGE_TO_USE_IN_SPEC,
+        RCUSCHEMAPREFIX, dbNamespace, 0, dbUrl),
+        String.format("Failed to create RCU schema for prefix %s in the namespace %s with "
+        + "dbUrl %s", RCUSCHEMAPREFIX, dbNamespace, dbUrl));
+
+    logger.info("DB image: {0}, FMW image {1} used in the test",
+        DB_IMAGE_TO_USE_IN_SPEC, FMWINFRA_IMAGE_TO_USE_IN_SPEC);
+
+    // install operator and verify its running in ready state
+    installAndVerifyOperator(opNamespace, domainNamespace);
+  }
+
+  /**
+   * Create a basic JRF dynamic cluster model in image domain.
+   * Verify Pod is ready and service exists for both admin server and managed servers.
+   * Verify EM console is accessible.
+   */
+  @Test
+  @DisplayName("Create JRF Dynamic Domain model in image")
```

Comment:
Change the Display Name  to "Create FMW Dynamic with Dynamic Cluster using model in image"

Reply:
changed